nextflow.config

```diff
@@ -10,14 +10,11 @@ params {
 
   // Boilerplate options
   name = false
-  multiqc_config = "$baseDir/assets/multiqc_config.yaml"
   email = false
   email_on_fail = false
-  maxMultiqcEmailFileSize = 25.MB
   plaintext_email = false
   monochrome_logs = false
   help = false
-  igenomes_base = "./iGenomes"
```

Comment:
For hlatyping we kept this (`igenomes_base = 's3://ngi-igenomes/igenomes/'`) and set `igenomes_ignore = true`. Not sure though if this is the preferred solution.

Reply:
done